cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/common/CubeSinkConfig.java

```diff
@@ -31,9 +31,7 @@
 
   @Name(Properties.Cube.DATASET_RESOLUTIONS)
   @Description("Aggregation resolutions to be used if a new Cube dataset " +
-    "needs to be created. See Cube dataset configuration " +
-    "details for more information : http://docs.cask.co/cdap/current/en/developers-manual/building-blocks/datasets/" +
-    "cube.html#cube-configuration")
+    "needs to be created. See Cube dataset configuration details for more information.")
```

Comment:
Perhaps say "See Cube dataset configuration details available at http://docs.cask.co for more information." ? I'm not sure...